site/docs/v1/tech/themes/index.adoc

```diff
@@ -9,27 +9,28 @@ description: Learn how to theme and the FusionAuth login pages (including forgot
 
 FusionAuth themes allow you to customize the OAuth2 / OpenID Connect login pages and other user workflows such as forgot password. In FusionAuth you may create one to many themes and assign a theme per tenant or application so that you can customize the user experience for different users.
 
-In this guide you will find a video demonstration, a list of each template available to you and some example code.
+To demonstrate what is possible with the FusionAuth feature we have created a video that illustrates how to make FusionAuth look like StackOverflow.
+
+video::_ro3jH5Xkgo[youtube,width=560,height=315]
+
+Here are the topics in this section:
+
+* link:examples[Examples]
+* link:helpers[Helpers]
+* link:localization[Localization]
+* link:template-variables[Template Variables]
+
```

Comment:
This could be a section maybe?

== How to Create, Update, and Troubleshoot Themes

Reply:
hmmm. Not sure. I think it's cleaner without the additional section.